tests/cases/fourslash/jsdocDeprecated_suggestion1.ts

```diff
@@ -0,0 +1,167 @@
+// @noUnusedLocals: false
+// @Filename: a.ts
+//// export namespace foo {
+////     /** @deprecated */
+////     export function faff () { }
+////     [|faff|]()
+//// }
+//// const [|a|] = foo.[|faff|]() 
+//// foo[[|"faff"|]]
+//// const { [|faff|] } = foo
+//// faff()
+//// /** @deprecated */
+//// export function bar () {
+////     foo?.[|faff|]()
+//// }
+//// foo?.[[|"faff"|]]?.()
+//// [|bar|]();
+//// /** @deprecated */
+//// export interface Foo {
+////     /** @deprecated */
+////     zzz: number
+//// }
+//// /** @deprecated */
+//// export type QW = [|Foo|][[|"zzz"|]]
+//// export type WQ = [|QW|]
+
+// @Filename: b.ts
+//// import * as f from './a';
+//// import { [|bar|], [|QW|] } from './a';
+//// f.[|bar|]();
+//// f.foo.[|faff|]();
+//// [|bar|]();
+//// type Z = [|QW|];
+//// type A = f.[|Foo|];
+//// type B = f.[|QW|];
+//// type C = f.WQ;
+//// type [|O|] = Z | A | B | C;
+
+goTo.file('a.ts')
+const ranges = test.ranges();
+
+verify.getSuggestionDiagnostics([

```

Comment:
I’ve forgotten—do normal error baselines include suggestions? They are a lot easier to read than the json format fourslash uses here, so I’d prefer them. 

Edit: after reading the test, the four slash readability is not that bad. But it’s still not as good as the error baselines.